tests/integration/lens.go

```diff
@@ -75,9 +75,34 @@ func getMigrations(
 
 		configs, err := db.LensRegistry().Config(s.ctx)
 		require.NoError(s.t, err)
-
-		// The order of the results is not deterministic, so do not assert on the element
```

Comment:
todo: I think this comment still adds value and should be preserved somewhere.  It is not obvious reading the code as to why the order of the elements is not asserted.